src/arch/xtensa/smp/idc.c

```diff
@@ -234,11 +234,12 @@ static void idc_cmd(struct idc_msg *msg)
  * \brief Handles received IDC message.
  * \param[in,out] data Pointer to IDC data.
  */
-static void idc_do_cmd(void *data)
+static uint64_t idc_do_cmd(void *data, uint64_t udelay)
 {
 	struct idc *idc = data;
 	int core = arch_cpu_get_id();
 	int initiator = idc->received_msg.core;
+	int ret = 0;
 
```

Comment:
Why introduce ret? just return 0 at the end.

Reply:
Updated